src/api/telephony/portability/document/telephony-portability-document.v6.service.js

```diff
@@ -0,0 +1,76 @@
+angular.module('ovh-api-services').service('OvhApiTelephonyPortabilityDocumentV6', ($cacheFactory, $resource, $window, $http) => {
+  const cache = $cacheFactory('OvhApiTelephonyPortabilityDocumentV6');
+  const queryCache = $cacheFactory('OvhApiTelephonyPortabilityDocumentV6Query');
+
+  const interceptor = {
+    response(response) {
+      cache.remove(response.config.url);
+      queryCache.removeAll();
+      return response.resource;
+    },
+  };
+
+  const docResource = $resource('/telephony/:billingAccount/portability/:id/document', {
+    billingAccount: '@billingAccount',
+    id: '@id',
+    documentId: '@documentId',
+  }, {
+    query: {
+      method: 'GET',
+      isArray: true,
+    },
+    create: {
+      method: 'POST',
+      interceptor,
+    },
+    getDocument: {
+      method: 'GET',
+      url: '/telephony/:billingAccount/portability/:id/document/:documentId',
+      isArray: false,
+    },
+    updateDocument: {
+      method: 'PUT',
+      url: '/telephony/:billingAccount/portability/:id/document/:documentId',
+      interceptor,
+    },
+    deleteDocument: {
+      method: 'DELETE',
+      url: '/telephony/:billingAccount/portability/:id/document/:documentId',
+      interceptor,
+    },
+    cors: {
+      method: 'POST',
+      url: '/telephony/:billingAccount/portability/:id/document/cors',
```

Comment:
This URL doesn't exist on the API.

Shouldn't be done via the `create` action?

Reply:
yes I removed it